rust/Cargo.toml

```diff
@@ -38,6 +38,9 @@ uuid = { version = "0.8.1", features = ["v4"]}
 rayon = "1.3.0"
 async-trait = "0.1.35"
 
+# optional dependencies
+influxdb = { version = "0.1.0", features = ["derive"], optional = true }
+
```

Comment:
Can you please add `chrono = "0.4.13"` here as well. We need this library to create the timestamps of the metric data points.

Reply:
sure!